client/components/confirmation-modal/index.tsx

```diff
@@ -11,9 +11,26 @@ import { HorizontalRule } from '@wordpress/primitives';
  */
 import './styles.scss';
 
-const ConfirmationModal = ( { children, actions, className, ...props } ) => (
+interface ConfirmationModalInput {
```

Comment:
I'd suggest using following (incl. naming conventions):

```
interface ConfirmationModalProps extends Modal.Props {
	actions: JSX.Element;
}
```

Reply:
I've been looking at how I can achieve this yesterday, didn't know about the `extends Modal.props` part, thank you!